src/main.rs

```diff
@@ -112,7 +135,7 @@ fn main() {
     let total_start = Instant::now();
     for package in config.get_app_packages() {
         config.reset_force();
-        analyze_package(package, &mut config, &mut benchmarks);
+        analyze_package(package, &mut config, &mut benchmarks)?;
```

Comment:
We should probably `chain_err()` this too, with a message saying something like "the application analysis failed". A chain like that would make the final backtrace really pretty.